tools/ShapeChecker/src/main/java/net/open_services/scheck/shapechecker/HttpHandler.java

```diff
@@ -143,9 +150,32 @@ else if (containsMatch(skipURIPatterns,httpUri.toString()))
     }
 
 
+    /**
+     * Hack to work around issue with Jena handling of Dublin Core redirects.
+     * @param httpUriOrig a URI
+     * @return if the input was a DCTerms URI, return a modified URI reflecting the several redirects
+     */
+    private static URI issue88(URI httpUriOrig)
+    {
+        Pattern dcterms = Pattern.compile("^http://purl\\.org/dc/terms/(.*)$");
+        URI httpUri = httpUriOrig;
+        Matcher m = dcterms.matcher(httpUri.toString());
```

Comment:
I think this match can be simply done with the startsWith method

Reply:
The startsWith method doesn't capture the string after the starting match, and I need that to fill in the rest of the URI.